app/import_functions.go

```diff
@@ -530,11 +530,13 @@ func (a *App) importUser(data *UserImportData, dryRun bool) *model.AppError {
 	if data.ProfileImage != nil {
 		file, err := os.Open(*data.ProfileImage)
 		if err != nil {
-			mlog.Error("Unable to open the profile image.", mlog.Any("err", err))
+			mlog.Warn("Unable to open the profile image.", mlog.Any("err", err))
```

Comment:
mlog.Err(err) ?   os.Open can only return an error or nil for err.

Reply:
Thanks, this escaped from me 👍